Makefile

```diff
@@ -1,41 +1,99 @@
+	$(eval export PORT=$(COLLECTOR_LISTEN_PORT))
 	./bin/paas-billing collector | ./scripts/colorize
 
-run-dev-api: bin/paas-billing run-dev-exports
+.PHONY: run-dev-api
+run-dev-api: dev-prerequisites
+	$(eval export VCAP_APPLICATION={"application_id":"17f5830c-01df-4201-bc66-d33f4b1b2244","application_name":"paas-billing-api","application_version":"461feda8-03ba-4ede-a5b5-04524147c621","instance_id":"0c59e331-e446-4ec2-aced-432b44397231","instance_index":0,"organization_id":"a0caff22-cc01-4370-bc36-592a4aba2dfd","organization_name":"admin","process_id":"22e75a85-cb20-4a51-806a-6e0aa5901364","process_type":"web","space_id":"efdd5187-536e-49d8-8b8c-55436515ada6","space_name":"billing"})
+	$(eval export PORT=$(API_LISTEN_PORT))
 	./bin/paas-billing api | ./scripts/colorize
 
-run-dev-exports:
+.PHONY: run-proxy-metrics
+run-dev-proxy-metrics: dev-prerequisites
+	$(eval export VCAP_APPLICATION=$(shell cf curl "/v3/apps/$$(cf app paas-billing-metrics-proxy --guid)/env" | jq -c '.application_env_json.VCAP_APPLICATION'))
+	$(eval export APP_NAMES=paas-billing-api,paas-billing-collector)
+	$(eval export PORT=$(PROXYMETRICS_LISTEN_PORT))
+
+	./bin/paas-billing proxymetrics | ./scripts/colorize
+
+.PHONY: run-dev-exports
+run-dev-exports: .cf-is-dev .cf-target-admin-billing
 	## Runs the application with local credentials
 	$(eval export CF_API_ADDRESS=${CF_API_ADDRESS})
 	$(eval export CF_CLIENT_ID=paas-billing)
-	$(eval export CF_CLIENT_SECRET=$(shell aws s3 cp s3://gds-paas-${DEPLOY_ENV}-state/cf-vars-store.yml - | awk '/uaa_clients_paas_billing_secret/ { print $$2 }'))
+	$(eval export CF_CLIENT_SECRET=$(shell cf env paas-billing-collector | awk '/CF_CLIENT_SECRET:/ { print $$2 }'))
 	$(eval export CF_CLIENT_REDIRECT_URL=http://localhost:8881/oauth/callback)
 	$(eval export CF_SKIP_SSL_VALIDATION=true)
 	$(eval export DATABASE_URL=${DATABASE_URL})
 	$(eval export APP_ROOT=${APP_ROOT})
+	$(eval export LISTEN_HOST=${LISTEN_HOST})
+	$(eval export LOG_LEVEL=${LOG_LEVEL})
 	@true
 
 .PHONY: test
-test: fakes/fake_usage_api_client.go fakes/fake_cf_client.go fakes/fake_event_fetcher.go fakes/fake_event_store.go fakes/fake_authorizer.go fakes/fake_authenticator.go fakes/fake_billable_event_rows.go fakes/fake_usage_event_rows.go fakes/fake_cf_data_client.go
+test: fakes
 	$(eval export TEST_DATABASE_URL=${TEST_DATABASE_URL})
 	$(eval export APP_ROOT=${APP_ROOT})
 	go run github.com/onsi/ginkgo/v2/ginkgo $(ACTION) -nodes=8 -r $(PACKAGE) -skip-package=acceptance_tests
 
-# .PHONY: gherkin_test
+
+.PHONY: gherkin_test
 gherkin_test: gherkin_test_lon gherkin_test_ie
 
-# .PHONY: gherkin_test_lon
+.PHONY: gherkin_test_lon
 gherkin_test_lon:
 	mkdir -p gherkin/features
 	cp ../paas-cf/config/billing/tests/eu-west-1_billing_rds_charges.feature gherkin/features/
```

Comment:
gherkin tests have the prod regions mixed up but appears as though prior to current changes